src/calculateRentalCost.js

```diff
@@ -28,7 +28,17 @@
  */
 
 function calculateRentalCost(days) {
-  // write code here
+  const PRICE_PER_DAY = 40;
+  const LONG_TERM = 7;
+  const SHORT_TERM = 3;
+  const LONG_TERM_DISCOUNT = 50;
+  const SHORT_TERM_DISCOUNT = 20;
+
+  const dayRentPrice = PRICE_PER_DAY * days;
```

Comment:
```suggestion
  const basicPrice = PRICE_PER_DAY * days;
```
Actually, it's not a day rent price. It's a price for all days without discounts